lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationFeedFormatter.cs

```diff
@@ -6,17 +6,13 @@
 
 namespace Microsoft.ServiceModel.Syndication
 {
+    using Microsoft.ServiceModel.Syndication.Resources;
     using System;
-    using System.Diagnostics;
     using System.Globalization;
-    using System.Runtime;
-    using System.Runtime.Serialization;
+    using System.Threading;
+    using System.Threading.Tasks;
     using System.Xml;
-    using System.Runtime.CompilerServices;
-    using Microsoft.ServiceModel.Syndication.Resources;
 
-    [TypeForwardedFrom("System.ServiceModel.Web, Version=3.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35")]
-    [DataContract]
     public abstract class SyndicationFeedFormatter
```

Comment:
Must have deleted by accident, re adding it.

---
In reply to: [126765791](https://github.com/dotnet/wcf/pull/2028#discussion_r126765791) [](ancestors = 126765791)